biotope/2018-06-05_biotopeVision.md

```diff
@@ -1,26 +1,24 @@
-# Biotope is a framework which enables you to develop scalabale, platform-agnostic and living frontend component ecosystems
+# Biotope is a framework which enables you to develop scalabale, platform-agnostic frontend component ecosystems
 
 // Insert short version
 
 Definition Ecosystem vs. Ecosystem Framework (was ist biotope).
 
 ## It helps developers to:
-- Build your UI components only once and use them together with vanilla javascript or the js framework of your choice, in single page applications as well as in traditional server side rendered cms projects.
+- Build your UI components once and use them together with vanilla javascript or the js framework of your choice, in single page applications as well as in traditional server side rendered cms projects.
 - Keep focus on developing components, the architecture and dev environment is provided by Biotope.
 - Easy start, love to use it.
-- Build real projects and design system styleguides with a single code source.
+- Build projects and design system styleguides with a single code source. 
 - Easily ensure high quality standards for the whole library even when developed by multiple suppliers.
 - Make it easy to work with a team of many developers, all providing components to the library.
 - Create your own ui library to choose from in your projects.
 
 
 ## It helps companies to:
 - Create a UI component library which can be maintained in a central place.
-- STILL all ui-components can be used independently in different projects.
+- Versioning enables you to use all ui-components independently in different projects.
```

Comment:
thanks @johanneskleiber as discussed i ll merge your commit and afterwards revert/rewrite this part.